### DISCUSSION TITLE
Week 5 Feelings and Ideas

### DISCUSSION THREAD

**Discussion by gabebott** [USU-ITLS-x265-23-summer/class-repo#119] [General]
FreeCodeCamp: I have personally struggled to get good explanation and forgiveness from this site. I feel like the exercises are good, but I want a little more feedback when I mess up, or some resources within the program that can help me learn instead of just leaving me guessing 10 versions of the same thing and searching other sites for explanations on the specific levels. I think that there are two sides of it. I like how challenging it is, and it makes it feel more rewarding when I get one right, but dang it can be tricky sometimes. 
 
Flexbox Froggy game: I had the opposite experience with this one. I think it was very simple to grasp, and the feedback was good in the fact that I could go back a few levels and see in the instructions what the specific functions of each property. I thought this game was fun, but it was a little too easy. Especially when I figured out I didn't even have to type and you could just click the properties. Fun game, but a little too easy.

Project Task
I liked the idea of making a form or survey with the HTML activity that we did. I think this kind of practice could be implemented into my ideas of creating a movie quote generator and a to-do list app. I think inputting information and getting a result from a database is something that I could use this type of coding for.

**Comment by metasean** [comment]
> just leaving me guessing 10 versions 

hint: if you've done the "Check your Code" 3 or more times, a "💬 Help" button is displayed below the test results.  The button shows a pop-up, where you can easily access all the forum posts that are directly related to the given task, or make a post of your own.  Typically, any problem that you run into, somebody else has already run into, and so you can find an existing post that helps you with whatever challenge you've run into.
